### PR TITLE
New Package: Intel oneTBB

### PR DESCRIPTION
**Pull request by craffael**
* I've followed [this guide](https://hunter.readthedocs.io/en/latest/creating-new/create/cmake.html)
  step by step carefully. **Yes**

### CODE REVIEW

File: cmake/projects/oneTBB/hunter.cmake [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    "https://github.com/oneapi-src/oneTBB/archive/v2021.5.0.tar.gz"
+    SHA1
+    71750727bd1436f4047342d0adb827c25d7bc2b0
+)
+
+hunter_cmake_args(
+  oneTBB
+  CMAKE_ARGS
+    TBB_TEST=OFF
+	TBB_STRICT=OFF
```
Comment (NeroBurner) [inline cmake/projects/oneTBB/hunter.cmake:27]: use spaces for indentation, please don't mix it up

```suggestion
    TBB_STRICT=OFF
```

File: cmake/projects/oneTBB/hunter.cmake [hunk 2]
```diff
@@ -0,0 +1,32 @@
+# Copyright (c) 2016-2020, Rahul Sheth, Ruslan Baratov
```
Comment (NeroBurner) [inline cmake/projects/oneTBB/hunter.cmake:1]: please update copyright

**Review by NeroBurner — CHANGES_REQUESTED** [review]
just some small nitpicks, thank you for the contribution

File: examples/oneTBB/CMakeLists.txt [hunk 3]
```diff
@@ -0,0 +1,19 @@
+# Copyright (c) 2016-2020, Rahul Sheth, Ruslan Baratov
```
[the remark this answers is not in the document]
Comment (craffael) [reply examples/oneTBB/CMakeLists.txt:1]: Do you want me to update the year (replace 2020 by 2022)? And should I do this change also in `examples/foo/CMakeLists.txt`? That way the template which is used in [this guide](https://hunter.readthedocs.io/en/latest/creating-new/create/cmake.html#create-example) would be up to date...

[the remark this answers is not in the document]
Comment (craffael) [reply examples/oneTBB/CMakeLists.txt:1]: btw. Thanks for the almost immediate review!

[the remark this answers is not in the document]
Comment (NeroBurner) [reply examples/oneTBB/CMakeLists.txt:1]: I'd say adding yourself and replace the year with 2022, something like the following (replace the name, that you want to have there)

```suggestion
# Copyright (c) 2022, craffael
```

If the guide is out of date a PR to fix that would be very much appreciated :pray:

[the remark this answers is not in the document]
Comment (NeroBurner) [reply examples/oneTBB/CMakeLists.txt:1]: you're welcome :)

[the remark this answers is not in the document]
Comment (craffael) [reply examples/oneTBB/CMakeLists.txt]: thx for the suggestion and the fast review! I think I'll make a separate PR for the guide.

File: cmake/projects/oneTBB/hunter.cmake (hunk 2 above)
Comment (craffael) [reply cmake/projects/oneTBB/hunter.cmake]: done.

**Review by NeroBurner — APPROVED** [review]
LGTM

yes please, a separate PR would be very good :)

**Comment by NeroBurner** [comment]
Released: https://github.com/cpp-pm/hunter/releases/tag/v0.24.5

**Pull request merged**